filters.json.example

```diff
@@ -288,5 +288,270 @@
         "Lugia":"False",
         "Ho-Oh":"False",
         "Celebi":"False"
+    },
+    "raids":{
+        "enabled":"True",
+        "filters": {
```

Comment:
Can we name this to 'eggs' instead of filters?  Filters doesn't seem quite appropriate as the terminology.